src/objects/Mesh.js

```diff
@@ -210,7 +212,12 @@ Mesh.prototype = Object.assign( Object.create( Object3D.prototype ), {
 
 				}
 
-				intersection.face = new Face3( a, b, c, Triangle.normal( vA, vB, vC ) );
+				var face = new Face3( a, b, c );
+
+				triangle.set( vA, vB, vC ).normal( faceNormal );
+				face.normal.copy( faceNormal );
```

Comment:
In any event, you can write your existing code like so:
```javascript
triangle.set( vA, vB, vC ).normal( face.normal );
```
Or, better yet:
```javascript
Triangle.normal( vA, vB, vC, face.normal );
```

Reply:
Much better!